src/Elliptic/Systems/Elasticity/FirstOrderSystem.hpp

```diff
@@ -33,64 +30,33 @@ struct Operand;
  *
- * \details This system formulates the Elasticity equation \f$\nabla T(x) =
- * f(x)\f$ on a euclidean space as the set of coupled first-order PDEs
+ * This system formulates the Elasticity equation \f$\nabla_i T^{ij}(x) =
+ * f^j(x)\f$. It introduces the symmetrical strain tensor S as an auxialiary
+ * variable which satisfies the ConstitutiveRelation \f$T^{ij}(x) = -Y^{ijkl}(x)
+ * S_{kl}(x)\f$ with the material-specific Youngs tensor Y. Written as a set of
+ * coupled first-order PDEs, we get
  *
  * \f[
- * \nabla \boldsymbol{T}(x) = \boldsymbol{f}(x) \\
- * -\frac{\nabla \boldsymbol{\xi}(x) + (\nabla \boldsymbol{\xi}(x))^T}{2} +
- * \boldsymbol{S}(x) = 0 \f]
- *
- * with the stress tensor T given by \f$T^{ij}(x) = -Y^{ijkl}S_{kl}(x)\f$,
- * \f$Y^{ijkl}\f$ being the material-specific Youngs tensor and the strain given
- * by \f$S_{kl}(x) = \partial_{(k}\xi_{l)}(x)\f$ as an auxiliary variable.
- *
- *
- * This scheme also goes by the name of _mixed_ or _flux_ formulation (see e.g.
- * \cite Arnold2002). The reason for the latter name is that we can write the
- * set of coupled first-order PDEs in flux-form
- *
- * for the primary equation
- *
- * \f[
- * \partial_i F^i_A + S_A = f_A(x)
- * * \f]
- *
- *  and respectively, for the auxiliary equation
- *
- * \f[
- * -\partial_i F^i_A + S_A = f_A(x)
+ * -\nabla_i (Y^{ijkl}(x) S_{kl}(x)) = f^j(x) \\
+ * -\nabla(_k \xi_l(x) + S_{kl}(x) = 0
  * \f]
  *
- * by choosing the fluxes and sources in terms of the system variables
- * \f$\xi_j(x)\f$ and \f$S_{kl}(x)\f$ as
+ * The fluxes and sources in terms of the system variables
+ * \f$\xi^j(x)\f$ and \f$S_{kl}(x)\f$ are given by
  *
  * \f{align*}
- * F^i_{\xi_j} &=  -Y^{ijkl}_{(\Chi, \\xi, S)} S_{kl}(x) \\
- * S_{\xi_j} &= 0 \\
- * f_{\xi_j} &= f^j(x) \\
+ * F^i_{\xi^j} &=  Y^{ijkl}_{(\xi, S)}(x) S_{kl}(x) \\
+ * S_{\xi^j} &= 0 \\
+ * f_{\xi^j} &= f^j(x) \\
  * F^i_{S_{kl}} &= \delta^{i}_{(k} \xi_{l)} \\
  * S_{S_{kl}} &= S_{kl} \\
  * f_{S_{kl}} &= 0 \text{.}
  * \f}
  *
- * Note that we use the system variables to index the fluxes and sources, which
- * we also do in the code by using DataBox tags.
- * Also note that we have defined the _fixed sources_ \f$f_A\f$ as those source
- * terms that are independent of the system variables.
- *
- * The strain \f$S_{kl}\f$ is treated on the same footing as the field
- * \f$\xi_k\f$ in this first-order formulation. This allows us to make use of
- * the DG architecture developed for coupled first-order hyperbolic PDEs in
- * flux-form, in particular the flux communication and lifting code. It does,
- * however, introduce auxiliary degrees of freedom that can be avoided in the
- * second-order (or _primal_) formulation. Furthermore, the linear operator that
- * represents the DG discretization for this system is not symmetric. This
- * property further increase the computational cost (see \ref LinearSolverGroup)
- * and is remedied in the second-order formulation.
+ * See Poisson::FirstOrderSystem
```

Comment:
how about:
"See \`Poisson::FirstOrderSystem\` for details on the first-order flux-formulation."